week-5-extended-project/team-space/seed.rb

```diff
@@ -0,0 +1,35 @@
+require 'csv'
+
+headers, *data = CSV.read("data.csv")
+
+file_agencies = File.open("seeds/agencies.sql", "w")
+file_customers = File.open("seeds/customers.sql", "w")
+file_sales = File.open("seeds/sales.sql", "w")
+file_tours = File.open("seeds/tours.sql", "w")
+
+tours = []
+agencies = []
+customers = []
+
+data.each do |row|
+  row = row.map do |r|
```

Comment:
What is `r`?

Reply:
Sorry, I just wanted to go through the array and add the single quote and I did not choose a good name for that variable.